sources/web/datalab/polymer/test/item-list-test.ts

```diff
@@ -119,16 +147,50 @@ describe('<item-list>', () => {
     const c = testFixture.$.header.querySelector('#selectAllCheckbox') as HTMLElement;
     c.click();
 
-    assert(JSON.stringify(testFixture.selectedIndices) === '[0,1,2]');
+    assert(JSON.stringify(testFixture.selectedIndices) === '[0,1,2,3,4]',
+        'all items should be in the selected indices array');
     assert(isSelected(0) && isSelected(1) && isSelected(2), 'all items should be selected');
```

Comment:
I'd like to have at least one place where `isSelected` is called on all elements to maximize its benefit.